src/rhoai_in_kind/__init__.py

```
@@ -0,0 +1,113 @@
+import subprocess
+import sys
+import time
+from typing import TYPE_CHECKING, Generator
+
+if TYPE_CHECKING:
+    from typing import Any, Callable
+
+
+def sh(
+    cmd: str, env: dict[str, str] | None = None,
+    input: str | None = None,
```

Ruff (0.16.1)

[error] Function argument `input` is shadowing a Python builtin

(A002)